### PR TITLE
numpy frontend tests - mgrid, ogrid

### PR DESCRIPTION
**Pull request by juliagsy**

### CODE REVIEW

File: ivy_tests/test_ivy/test_frontends/test_numpy/test_creation_routines/test_numerical_ranges.py [hunk 1]
```diff
@@ -193,6 +217,67 @@ def test_numpy_meshgrid(
...
 
-# TODO
 # mgrid
+@handle_cmd_line_args
+@given(range=_get_range_for_grid())
+def test_numpy_mgrid(
+    range,
+    fw,
+):
+    ivy.set_backend(fw)
```
Comment (djl11) [inline ivy_tests/test_ivy/test_frontends/test_numpy/test_creation_routines/test_numerical_ranges.py:227]: this line should not be needed. It should be handled by `@handle_cmd_line_args`, are you sure it's required?

File: ivy_tests/test_ivy/test_frontends/test_numpy/test_creation_routines/test_numerical_ranges.py [hunk 2]
```diff
@@ -193,6 +217,67 @@ def test_numpy_meshgrid(
...
+    elif start:
+        ret = mgrid[start:]
+        ret_np = np_mgrid[start:]
+    else:
+        ret = mgrid[:stop]
+        ret_np = np_mgrid[:stop]
+    ret = helpers.flatten_and_to_np(ret=ret)
+    ret_np = helpers.flatten_and_to_np(ret=ret_np)
+    helpers.value_test(ret_np_flat=ret, ret_np_from_gt_flat=ret_np, rtol=1e-03)
+    ivy.unset_backend()
```
Comment (djl11) [inline ivy_tests/test_ivy/test_frontends/test_numpy/test_creation_routines/test_numerical_ranges.py:250]: likewise

File: ivy_tests/test_ivy/test_frontends/test_numpy/test_creation_routines/test_numerical_ranges.py [hunk 3]
```diff
@@ -193,6 +217,67 @@ def test_numpy_meshgrid(
...
+
+
 # ogrid
+@handle_cmd_line_args
+@given(range=_get_range_for_grid())
+def test_numpy_ogrid(
+    range,
+    fw,
+):
+    ivy.set_backend(fw)
```
Comment (djl11) [inline ivy_tests/test_ivy/test_frontends/test_numpy/test_creation_routines/test_numerical_ranges.py:260]: likewise

File: ivy_tests/test_ivy/test_frontends/test_numpy/test_creation_routines/test_numerical_ranges.py [hunk 4]
```diff
@@ -193,6 +217,67 @@ def test_numpy_meshgrid(
...
+    elif start:
+        ret = ogrid[start:]
+        ret_np = np_ogrid[start:]
+    else:
+        ret = ogrid[:stop]
+        ret_np = np_ogrid[:stop]
+    ret = helpers.flatten_and_to_np(ret=ret)
+    ret_np = helpers.flatten_and_to_np(ret=ret_np)
+    helpers.value_test(ret_np_flat=ret, ret_np_from_gt_flat=ret_np, rtol=1e-03)
+    ivy.unset_backend()
```
Comment (djl11) [inline ivy_tests/test_ivy/test_frontends/test_numpy/test_creation_routines/test_numerical_ranges.py:283]: likewise

**Review by djl11 — CHANGES_REQUESTED** [review]

File: ivy_tests/test_ivy/test_frontends/test_numpy/test_creation_routines/test_numerical_ranges.py (hunk 1 above)
Comment (juliagsy) [reply ivy_tests/test_ivy/test_frontends/test_numpy/test_creation_routines/test_numerical_ranges.py]: oh right, I just checked and it's done with `f.use`, just removed them! thanks!

**Pull request merged**